src/instructlab/config/init.py

```diff
@@ -183,6 +202,60 @@ def init(
     )
 
 
+def hw_auto_detect() -> tuple[str | None, int | None, int | None, _train, bool]:
+    # Third Party
+    import torch
+
+    profiles = render_profiles()
+    gpus = 0
+    total_vram = 0
+    train = _train()
+    chosen_vram = None
+    chosen_profile_name = None
+    previous_vram = None
+    # try nvidia
+    if torch.cuda.is_available():
+        click.echo("Detecting Hardware...")
+        gpus = torch.cuda.device_count()
+        for i in range(gpus):
+            properties = torch.cuda.get_device_properties(i)
+            total_vram += properties.total_memory  # memory in B
+    vram = convert_bytes_to_proper_mag(total_vram)[0]
+    vram = 89
```

Comment:
Looks like an unintentional double assign, debugging?

Reply:
thanks Jason, totally missed this!!! Leftover debugging 😅